integration_tests/support/commands.ts

```diff
@@ -1,4 +1,6 @@
-Cypress.Commands.add('signIn', (options = { failOnStatusCode: true }) => {
```

Comment:
odd declaration here, it is defining `failOnStatusCode` as a type called `true` rather than a `boolean` type.  Since we just pass through the options anyway better to use the cypress type